yente/app.py

```diff
@@ -44,32 +42,14 @@ async def lifespan(app: FastAPI) -> AsyncGenerator[None, None]:
     await close_provider()
 
 
-async def request_middleware(
+async def catch_exception_middleware(
```

Comment:
just an idea, don't feel strongly about it

```suggestion
async def json_exception_middleware(
```